Dashboard/app/js/lib/components/stats/StatsLists.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class StatsList extends React.Component {
+  renderNoStats = () => {
+    return (
+      <div className="no-stats">
+        <p>No stats generated yet</p>
+        <p>Click {'"Export stats"'} to get started</p>
+      </div>
+    );
+  };
+
+  renderStat = stat => {
+    return (
+      <a href={stat.url} key={stat.id}>
+        <div className="generated-stat">
+          <div className="stats-details">
+            <span className="stat-icon" />
+            <div>
+              <span className="date">
+                {stat.startDate} - {stat.endDate} Submissions
```

Comment:
What about the cases where one of either start or end date or even both are left out?  For now all we show is:

![dates-not-provided](https://user-images.githubusercontent.com/533336/68046106-583de700-fcdb-11e9-9b9b-b8280bb46994.png)

Reply:
@janagombitova and @Kiarii any help here?

Reply:
Answering this not seeing the implementation yet: 

Defining the dates is optional so there is a high chance users will want to get the Stats from the beginning of time till today. I would suggest that: 
* if the start date is not defined we simply use 'from the beginning of time' that keeps it clear, generic and is a term used by other household tools so users know. 
* if the end date is not defined, then we should always use the current date (today) in the date format

Reply:
we had mockups illustrating how the date picking would work by default; without the user having to enter anything; https://invis.io/9NUGWCM82MK#/389609091_0-_Stats_-_HOME_-_ES

Reply:
@Kiarii can't find the mock-up that points to when no date is added